backend/src/server.ts

```diff
@@ -63,6 +78,37 @@ export const context = async (options) => {
   }
 }
 
+export const loggerPlugin = {
+  requestDidStart(requestContext) {
+    const isIntrospectionQuery = requestContext.request.operationName === 'IntrospectionQuery'
+    const qID = randomBytes(4).toString('hex')
+    if (!isIntrospectionQuery) {
+      const logRequest = ['Apollo Request', qID, requestContext.request.operationName]
+      logRequest.push(JSON.stringify(requestContext.request.query))
+      if (requestContext.request.variables) {
+        const variables = cloneDeep(requestContext.request.variables)
+        if (variables.password) variables.password = '***'
```

Comment:
I would like to see this software-tested before we merge. You can expect `context.logger` to be called with an obfuscated password on mutation `login`.